src/resolve.rs

```diff
@@ -55,12 +55,18 @@ impl AdmonitionMeta {
 
 /// Make the first letter of `input` upppercase.
 ///
-/// source: https://stackoverflow.com/a/38406885
-fn ucfirst(input: &str) -> String {
-    let mut chars = input.chars();
-    match chars.next() {
-        None => String::new(),
-        Some(f) => f.to_uppercase().collect::<String>() + chars.as_str(),
+/// We special case a few words to make them look nicer (e.g. "tldr" -> "TL;DR" and "faq" -> "FAQ").
+fn as_title_case(input: &str) -> String {
```

Comment:
nit: `as_title_case` isn't actually correct here, given the special casing.

I think renaming `ucfirst` to `as_title_case` is good to improve readability, but this new function with the special cases should be something more semantic like `directive_default_title`?

Reply:
Title case would be every first letter of each word uppercase. I called it format_directive_title in https://github.com/tommilligan/mdbook-admonish/pull/154/commits/d5bdde1f5cee6d3a8331eba82dcdc4cc8e5a02ea